megamek/src/megamek/client/ui/baseComponents/AbstractButtonDialog.java

```diff
@@ -42,6 +42,7 @@
  * This is directly tied to MekHQ's AbstractMHQButtonDialog, and any changes here MUST be verified
  * there.
  */
+@SuppressWarnings("serial") // Same-version serialization only (See Swing base classes)

```

Comment:
This is completely IDE setup dependent. My IDE (IDEA) gives me the exact opposite warning, mentioning that the suppression is redundant. Further, we do not employ serialization in any of our components.

Reply:
VSCode complains about this.

Reply:
I will revert this set of change.  I had a suspicion that these were IDE specific.  I can change my IDE settings to accommodate. I don't want to simply move the warning from one environment to another.  :)